### DISCUSSION TITLE
Deta modern runtime support

### DISCUSSION THREAD

**Discussion by chiboreache** [fastapi/fastapi#6386] [Questions] [answered]
### Operating System

Arch Linux

### Python Version

Python 3.10.4

### Additional Context

i know it's free and whatever, but it's a shame to force some service in docs which is running on outdated tech..

```sh
Available runtimes:
        Python: python3.7, python3.8, python3.9
```

and moreover they seems don't care, i tried to ask about update in Slack, but with no luck

**Comment by Kludex** [comment]
And... How is this related to FastAPI?

**Comment by chiboreache** [comment]
https://fastapi.tiangolo.com/deployment/deta/

**Comment by tiangolo** [comment, marked as answer]
So, you're suggesting to remove docs? I'll pass on that, but thanks for the interest.

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
